components/SplitPane/common/src/commonMain/kotlin/org/jetbrains/compose/splitpane/SplitPane.kt

```diff
@@ -0,0 +1,78 @@
+package org.jetbrains.compose.splitpane
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+data class MinimalSizes(
+    val firstPlaceableMinimalSize: Dp,
+    val secondPlaceableMinimalSize: Dp
+)
+
+@Composable

```

Comment:
Please provide documentation for all public API functions.

Reply:
1

Reply:
java doc commentary added to all public API.